geode-core/src/test/java/org/apache/geode/internal/cache/DistributedRegionTest.java

```diff
@@ -260,4 +266,132 @@ public void validateAsynchronousEventDispatcherShouldThrowExceptionWhenDispatche
         .hasMessage("Parallel Gateway Sender " + senderId
             + " can not be used with replicated region " + regionPath);
   }
+
+  @Test
+  public void hasSeenEventDoseNotFindAndSetVersionTagIfFoundInEventTrackerAndVersionTagIsSet() {
```

Comment:
Typo here and in other tests added, should be "DoesNot" rather than "DoseNot"

Reply:
Good catch!